### ISSUE TITLE
Possible to cache.set an array of JSON objects and get back an array?

### ISSUE THREAD

**Issue by xxkenta** [AkashBabu/redis-json#8]
As is, calling cache.set on an array of JSON objects like this:

```
[
  {
    id: 1,
    name: "John"
    age: 22
  },
  {
    id: 2,
    name: "James"
    age: 24
  }
]
```
Works, however when you cache.get it returns like this:
```
{
  '0': {
    id: '1',
    name: 'John',
    age: '22'
  },
  '1': {
    id: '2',
    name: 'James'
    age: '24'
  }
}
```
As is, I can revert this returned Object of objects by running something like this:

```
let result= await cache.get(key)
let array = []
for(let i = 0; i < Object.keys(reslt).length; i++){
    array.push(result[i])
}
```

I suppose it's not an issue but perhaps something that could be improved upon?

Thanks

**Comment by AkashBabu** [comment]
Thanks for the second bug @xxkenta . Will fix it by today or tom.

**Comment by AkashBabu** [comment]
Hi @xxkenta , this bug has been fixed in v4.0.0 🎉
Please update your library and let me know if you find any issues.
